test/x86/svml.c

```diff
@@ -10648,6 +10648,1440 @@ test_simde_mm512_div_epu64(SIMDE_MUNIT_TEST_ARGS) {
   return 0;
 }
 
+static int
+test_simde_mm_exp_ps (void) {
```

Comment:
This is the cause of the emscripten failures.  The functions should take `SIMDE_MUNIT_TEST_ARGS`, not `void`.  It was fixed in skel.c on Thursday, you probably just haven't updated the checkout you're using to generate these.

Reply:
Thanks! I have updated it.